src/layouts/Authenticated/LeftNav/index.tsx

```diff
@@ -35,6 +37,7 @@ const LeftNav: SFC = ({className}) => {
   return (
     <S.Container className={className}>
       <S.Top>
+        {/* TODO: replace these hardcoded paths with those in constants/paths.ts */}
```

Comment:
Added some of these TODO comments.
IMO, I think we should not hardcode paths, as they can scatter in the codebase, and if in future, we need to change a URL, we have to find and change ALL it's occurrences, which is not ideal.

Instead, if we are using a constant, then changing URL would only be a single line change.

@buckyroberts Let me know what you think. Please feel free to revert back on this